import/popups/LocationDialog.qml

```diff
@@ -17,17 +18,42 @@ UserPrompt {
 
     property string host
 
+    property alias rememberValue: remember.checked
+
     //: Allow the server to use location
     //% "Allow"
     acceptText: qsTrId("sailfish_components_webview_popups-he-accept_location")
 
-    Label {
-        x: Theme.paddingLarge
-        width: parent.width - Theme.paddingLarge * 2
-        //: The site that wants know user location
-        //% "The site '%1' wants to use your current location."
-        text: qsTrId("sailfish_components_webview_popups-la-location_requested").arg(host)
-        wrapMode: Text.WordWrap
-        color: Theme.highlightColor
+    //: Deny the server to use location
+    //% "Deny"
+    cancelText: qsTrId("sailfish_components_webview_popups-he-deny_location")
+
+    Column {
+        width: parent.width
+        spacing: Theme.paddingMedium
+
+        Label {
+            x: Theme.horizontalPageMargin
+            width: parent.width - x * 2
+
+            //: %1 is the site that wants know user location
+            //% "Allow %1 to use your location?"
+            text: qsTrId("sailfish_components_webview_popups-la-location_request")
+                        .arg("<font color=\"" + Theme.rgba(Theme.primaryColor, Theme.opacityHigh) + "\">" + host + "</font>")
```

Comment:
Please remove the underlining so that this is not mixed with actual links.

Just highlightColored text implying that special formatting is not needed either.

Reply:
Meant that let's have the whole text with highlightColor.